source/includes/language-compatibility-table-node.rst

```diff
@@ -15,7 +16,22 @@
      - Node.js v0.12.X
      - Node.js v0.10.X
      - Node.js v0.8.X
-   * - 5.X
+
+   * - >= 5.6.X
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     -
+     -
+     -
+     -
+     -
+     -
+     -
+     -
+   * - 5.0.0 to 5.5.X
```

Comment:
I: since this entry is longer than the others, it wraps to the next line and changes the uniformity of the table.

S: would it be possible to make the first column wider?

Reply:
Unfortunately, we can't control the width for this (weights only work when all the columns in the table fit without wrapping. Not sure what we can change the text to without introducing a new formatting standard such as using monospace.